src/views/Detail.vue

```diff
@@ -0,0 +1,47 @@
+<template>
+  <v-container>
+    <v-snackbar v-model="snackbar" :timeout="4000" top color="error">
+      <span>This track Id: {{ trackId }}, is invalid </span>
+      <v-btn dark color="white" class="black--text" @click="snackbar = false">Close</v-btn>
+    </v-snackbar>
+    <v-row dense>
+      <v-col cols="12">
+        <AppDetail :trackDetail="trackDetail" />
+      </v-col>
+    </v-row>
+  </v-container>
+</template>
+
+<script>
+import AppDetail from '@/components/app-detail/app-detail.vue';
+import api from '@/api';
+
+export default {
+  name: 'Detail',
+  components: {
+    AppDetail,
+  },
+  data() {
+    return {
+      trackDetail: {},
+      snackbar: false,
+      trackId: 0,
+    };
+  },
+  async mounted() {
+    this.trackId = Number(this.$route.params.id);
+    let result = {};
+    // eslint-disable-next-line no-restricted-globals
+    if (!isNaN(this.trackId)) {
```

Comment:
what about refactoring the readability? something like ...

`if (isNaN(this.trackId)) {
      this.snackbar = true;
      return;
    }

    this.trackDetail = await api.lookup(this.trackId);
    
    if (isEmpty(result)) {
      this.snackbar = true;
    }`

Reply:
done